config/config_test.go

```diff
@@ -31,6 +31,7 @@ var readTests = []struct {
 		"port":            8047,
 		"profiles":        []string{"default", "termserver"},
 		"session-timeout": 42,
+		"welcome-message": "exterminate!",
```

Comment:
I always appreciate a dalek reference, but "these are the voyages" was better. :laughing: